third_party/bigframes_vendored/pandas/core/frame.py

```diff
@@ -1795,6 +1814,10 @@ def iterrows(self):
             A    1
             B    4
             Name: 0, dtype: object
+
+        Yields:
```

Comment:
I'm curious. Does this render OK when you stage it?

If not, we might have to change to `Returns: Iterable[Tuple]: ...`

Reply:
I checked the staged doc, it didn't render. I switched it to `Returns: Iterable[Tuple]....`.